packages/ripple-nuxt-tide/lib/core/tide-helper.js

```diff
@@ -124,7 +126,9 @@ export const getPageHeadConfig = ({
       { name: 'twitter:image:alt', hid: 'hid:image:alt', content: imageAlt },
       // Custom page meta
       { name: 'sitesection', content: siteSectionName },
-      { name: 'content-type', content: pageType && pageType.replace('node--', '') }
+      { name: 'content-type', content: pageType && pageType.replace('node--', '') },
+      // Robots meta
+      robotsNoIndex ? { name: 'robots', content: robotsNoIndex } : ''
```

Comment:
👍

Reply:
Do we have some unit test around this function? Sorry probably me unfamiliar with the process 😬

Reply:
@rubyllamadora I don't think we have unit test placed for this function :disappointed:

Reply:
Hey @MdNadimHossain @rubyllamadora I didn't add unit test for this one. But I should. Sorry for that. All the unit tests are in https://github.com/dpc-sdp/ripple/tree/master/packages/ripple-nuxt-tide/test/unit
And for tide-helper tests, all in https://github.com/dpc-sdp/ripple/blob/master/packages/ripple-nuxt-tide/test/unit/tide.test.js.

Can you add one for this in this PR? Cheers!

Reply:
@tim-yao @rubyllamadora I have added the test.